packages/babel-plugin-react-i18n/src/test/BabelPluginReactI18n.test.ts

```diff
@@ -0,0 +1,87 @@
+import * as path from 'path';
+import {transformFile} from '@babel/core';
+import BabelPluginReactI18n from '../BabelPluginReactI18n';
+
+function runFixture(fixturePath: string) {
+  return new Promise<string>((resolve, reject) => {
+    transformFile(
+      fixturePath,
+      {
+        plugins: [BabelPluginReactI18n],
+        sourceRoot: __dirname,
+        filename: fixturePath,
+        filenameRelative: path.relative(__dirname, fixturePath),
+      },
+      (err, result) => {
+        if (err) {
+          reject(err);
+        } else {
+          // eslint-disable-next-line typescript/no-non-null-assertion
+          resolve(result!.code!);
+        }
+      },
+    );
+  });
+}
+
+describe('babel-pluin-react-i18n', () => {
+  it('injects arguments when adjacent translations exist', async () => {
```

Comment:
I'd consider adding a test to make sure it doesn't do anything to components without the decorator.